src/main/java/org/gridsuite/study/server/service/StudyService.java

```diff
@@ -2138,13 +2140,13 @@ public void invalidateNodeTree(UUID studyUuid, UUID nodeUuid, UUID rootNetworkUu
 
         InvalidateNodeInfos invalidateNodeInfos = networkModificationTreeService.invalidateNodeTree(nodeUuid, rootNetworkUuid, invalidateTreeParameters);
         invalidateNodeInfos.setNetworkUuid(rootNetworkService.getNetworkUuid(rootNetworkUuid));
-
         CompletableFuture<Void> cf = deleteInvalidationInfos(invalidateNodeInfos);
         if (blocking) {
             cf.join();
         }
 
         if (!networkModificationTreeService.isRootNode(nodeUuid)) {
+            rootNetworkNodeInfoService.clearNodeExportNetworks(nodeUuid, rootNetworkUuid);
```

Comment:
You can remove this call after you did https://github.com/gridsuite/study-server/pull/899/changes#r2653235285